### ISSUE TITLE
Relocation error trying to run on Ubuntu 19. Works fine in Ubuntu 18.

### ISSUE THREAD

**Issue by jaclynt** [JonathonReinhart/staticx#91]
Hi, I have a program made using pyinstaller (with `--onefile` flag) on Ubuntu 18. I then used staticx for it and it worked great running on Ubuntu 18 and also Debian 9. However, when I tried to run the same program in Ubuntu 19, I get the following error:

```
/tmp/staticx-AmMmbC/.staticx.prog: relocation error: /lib/x86_64-linux-gnu/libnss_files.so.2: symbol __libc_readline_unlocked version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```

Any ideas or more information I could provide that would be helpful?

**Comment by JonathonReinhart** [comment]
It looks like this may be related to #86.

**Comment by JonathonReinhart** [comment]
Hi @jaclynt.

Version [`v0.11.0`](https://github.com/JonathonReinhart/staticx/releases/tag/v0.11.0) was just released which contains #139, a major fix for NSS. With this version, you shouldn't run into this type of issue any more.

I'm confident it will fix the problem you were seeing, so I'm going to close this issue.

Please try it out and report back! Thanks for the interest in my project.
